extensions/oplogPopulator/OplogPopulatorMetrics.js

```diff
@@ -33,6 +33,10 @@ class OplogPopulatorMetrics {
             help: 'Total number of buckets per connector',
             labelNames: ['connector'],
         });
+        this.retainedBuckets = ZenkoMetrics.createGauge({
+            name: 's3_oplog_populator_connector_retained_buckets',
+            help: 'Total number of buckets still listened to by immutable connectors despite intended removal',
```

Comment:
nit: this is not the "total number" since it is a gauge.
More like the "current number"